config/application.rb

```diff
@@ -128,5 +128,7 @@ class Application < Rails::Application
                                                   authentication: ArchiveConfig.SMTP_AUTHENTICATION
                                                 })
     end
+
+    config.active_support.disable_to_s_conversion = true
```

Comment:
New Rails 7 parameter, but per the docs it needs to be put here and not in the overrides file